test/resource/user_test.rb

```diff
@@ -35,28 +35,58 @@
         .with(:get,
               "/v1/admin/auth/user/get",
               {
-                params: { issuer: issuer }, headers: {}
+                params: { issuer: issuer, wallet_type: MagicAdmin::Resource::WalletType::NONE }, headers: {}
               })
 
       subject.get_metadata_by_issuer(issuer)
     end
   end
 
+  context "#get_metadata_by_issuer_and_wallet" do
+    it "send request with options" do
```

Comment:
I see that there's some code duplicated between tests. ```#get_metadata_by_issuer_and_wallet``` and ```#get_metadata_by_issuer```, both have this for example:

```ruby
allow(MagicAdmin::Util).to receive(:headers)
        .with(magic.secret_key)
        .and_return({})
 ```

This is not urgent, as the tests are correct and work as intended, but maybe that duplicated code can be extracted to a ```before(:each)``` or ```before(:all)```. This same thing happes on ```#get_metadata_by_public_address``` and ```#get_metadata_by_public_address_and_wallet```

This would require to restructure the tests, tho. So not gonna request a change here, just something to keep in mind.

Reply:
Good point! I added a before(:each) to places it made sense and shaved off a few lines. Thanks for the rec!